landing.css

```diff
@@ -0,0 +1,126 @@
+@import "./base.css";
+@import "./utility.css";
+
+/* Navigation bar */
+
+.brand-name{
+    font-size: 2rem;
+    padding-bottom: 0.5rem;
+}
+.brand-logo{
+    margin-left: 1.6rem;
+    margin-top:1.2rem;
+}
+.nav-bar-hero{
+    display:flex
+}
+.nav-header{
+    background-color: var(--secondary-color) ;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: var(--text-white-color);
+}
+
+.nav-bar-tabs{
+    display: flex;
+    justify-content: space-around;
+    width:15rem;
+}
+.nav-text{
+    margin-top: 2rem;
+    font-weight: 600;
+}
+.nav-tab-text{
+    color:var(--primary-color);
+}
+.nav-tab-text:hover{
+cursor: pointer;
```

Comment:
Take care of consistent spacing in your code
Use formatter like Prettifier

Reply:
> Take care of consistent spacing in your code Use formatter like Prettifier

Used forced formatter for spacing now